agent/agent-filtering.md

```diff
@@ -0,0 +1,39 @@
+# Filter interface
```

Comment:
I think it is very important to clarify whether ordering is relevant for filters or not.